tests/foreman/longrun/test_inc_updates.py

```diff
@@ -0,0 +1,338 @@
+"""Tests for the Incremental Update feature"""
+
+from datetime import date, timedelta
+from fauxfactory import gen_alpha
+from nailgun import entity_mixins
+from nailgun.entities import (
```

Comment:
The majority of usage in this module is entities (I use 13+ entities) so calling them entities.* in each call will make the code redundant and boring.